src/Microsoft.AspNet.Mvc.Razor/LanguageViewLocationExpander.cs

```diff
@@ -3,13 +3,14 @@
 
 using System;
 using System.Collections.Generic;
-using Microsoft.AspNet.Mvc;
-using Microsoft.AspNet.Mvc.Razor;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
 
-namespace RazorWebSite
+namespace Microsoft.AspNet.Mvc.Razor
 {
     /// <summary>
-    /// A <see cref="IViewLocationExpander"/> that replaces adds the language as an extension prefix to view names.
+    /// A <see cref="IViewLocationExpander"/> that adds the language as an extension prefix to view names.
```

Comment:
Might want to also mention that the 'language' in practice is coming from the `HttpContext`